loopchain/blockchain/blocks/v1_0/block_factory.py

```diff
@@ -60,7 +60,7 @@ async def create_data(self, data_number: int, prev_id: bytes, epoch_num: int, ro
         block_builder.prev_votes = prev_votes
 
         pre_invoke_response: 'PreInvokeResponse' = self._invoke_pool.prepare_invoke(
-            block_height=data_number,
+            block_height=data_number-1,
```

Comment:
Can you explain why pre_invoke height is changed?

Reply:
It will be recovered.